Tests/TestProjectGenerator/Reqnroll.TestProjectGenerator/ProjectBuilder.cs

```diff
@@ -372,6 +376,19 @@ private void ConfigureMSTest()
             }
         }
 
+        private void ConfigureMSTest4()
```

Comment:
Obviously I don't know about that part. It seems this is used during new Reqnroll project.

Reply:
This is actually used by the SystemTests and the "Specs" project to generate sample MsTest v4 projects that we can use for system (end-to-end) testing. So this is only used for our intrnal tests, not by the end users. The new projects are created based on the templates in the Visual Studio extension or the `Reqnroll.Templates.DotNet` project.

Reply:
Aah. Okay. That also make sense. I think it potentially too fragile, but that’s entirely other question unrelated to MsTest4. That’s for explanation